engines/python/setup/djl_python/transformers_neuronx_scheduler/optimum_neuron_scheduler.py

```diff
@@ -178,7 +178,7 @@ def make_generations(
         return generation, finish_reason
 
     def _generate_token(self, inputs: GenerationInputs,
-                        prefill: Optional[bool]) -> List[Generation]:
+                        prefill: Optional[bool] = None) -> List[Generation]:
```

Comment:
Is this the error for do_sample?

Reply:
It is the error for the unit test with the tiny models, I can move it out into a separate PR if you would like. I forgot that I had added that fix here.

Reply:
No that's okay